### PR TITLE
CI: build all packages via GitHub action

### PR DESCRIPTION
**Pull request by aparcar**
Signed-off-by: Paul Spooren <mail@aparcar.org>

### CODE REVIEW

**Comment by aparcar** [comment]
This replaces travis or whatever is currently used with GitHub Actions. The roles should actually be split to offer per branch packages. Currently it's stored on my server, however I'd be happy to change that to any other server which should be used for LibreMesh.

**Comment by ilario** [comment]
Comparing this to https://github.com/libremesh/network-profiles/blob/master/.github/workflows/build.yml (which works perfectly, congratulations) the only thing missing is the EXTRA_FEEDS line.

**Comment by aparcar** [comment]
Where should the binaries be hosted? Some S3 storage, a GitHub/GitLab page, some rsync server?

File: packages/ubus-lime-utils/files/usr/libexec/rpcd/lime-utils [hunk 1]
```diff
@@ -68,15 +68,13 @@ local function get_node_status(msg)
...
     local stations = {}
     for _, iface  in ipairs(ifaces) do
-        if iface.name then
-            iface_stations = conn:call("lime-openairview", "get_stations", {device = iface.name}).stations
-            if iface_stations then
-                for _, station in pairs(iface_stations) do
-                    station['iface'] = iface.name
-                    table.insert(stations, station)
-                end
+        iface_stations = conn:call("lime-openairview", "get_stations", {device = iface}).stations
```
Comment (ilario) [inline packages/ubus-lime-utils/files/usr/libexec/rpcd/lime-utils:71]: This looks unrelated to the PR abount Github actions. Am I wrong?

**Review by ilario — COMMENTED** [review]
There is an edit to a file that looks unrelated to the PR. Apart from that, it looks ready to merge for me. We need someone with servers control to comment on where to host the files. Many many thanks to @aparcar  for temporarily hosting the files!

**Review by ilario — APPROVED** [review]
Looks perfect to me.
We should just decide where to host the binary repository.
We can merge this also before making the hosting decision and then change it (in the meanwhile it would be on @aparcar server), no?
@aparcar proposed to use Github pages, which would be ok for me.
Can we host the files on some of LibreMesh's servers? @nicoechaniz @altergui @nicopace @spiccinini @G10h4ck @p4u

**Comment by ilario** [comment]
Reminder, we will have to update also the repositories location here: 
https://github.com/libremesh/lime-packages/blob/903655831d74098ea2eba63da36d905636c6d4d0/packages/lime-system/files/etc/uci-defaults/92_add-lime-repos#L25-L31

**Review by spiccinini — APPROVED** [review]
Great! How is the signing being handled?

**Comment by aparcar** [comment]
Not yet, I have to add that to the gh-sdk-action.

**Comment by ilario** [comment]
@aparcar feel free to merge yourself, once you consider that the PR is ready :)

**Comment by aparcar** [comment]
I'll figure out the signing and then merge it.

**Comment by ilario** [comment]
The builds are failing due to the set-env in
https://github.com/libremesh/lime-packages/blob/f06b3dfb5b909bd96ce099ca8b660e48be0334e8/.github/workflows/build.yml#L31

which has been deprecated, see https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/

@aparcar do you have idea how to fix?

**Comment by spiccinini** [comment]
Starting to try to fix this at https://github.com/libremesh/lime-packages/pull/818
